### ISSUE TITLE
Add accessibility attributes

### ISSUE THREAD

**Issue by saibotsivad** [saibotsivad/svelte-progress-bar#8]
I've been reading through some ARIA docs and realized this component should probably just be a [`progress`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress) element, but if not it at least needs the [`role=progressbar`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/roles/progressbar_role) attribute.

In addition, I need to read through the docs, but there are some ARIA properties (labels, `valuenow`, etc) that should be set, and some guidance documented on the proper way to place this in a page for screen readers.
